ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -417,8 +417,8 @@ else
     export E2E_RUN_TAGS="$E2E_RUN_TAGS and $CUCUSHIFT_FORCE_SKIP_TAGS"
 fi
 date --utc
-oc version --client --output='yaml' || true
 set_cluster_access
+oc version --client --output='yaml' || true
```

Comment:
The change is unnecessary because with `--client` there is no outgoing traffic.

Reply:
Not a merge blocker.

Reply:
I just want to keep all code have same rule -- put set_cluster_access before all oc command.